packages/bot-web-ui/src/components/run-panel/run-panel.jsx

```diff
@@ -90,7 +90,7 @@ const DrawerContent = ({ active_index, is_drawer_open, setActiveTabIndex, ...pro
                     <Transactions is_drawer_open={is_drawer_open} />
                 </div>
                 <div id='db-run-panel-tab__journal' label={localize('Journal')}>
-                    <Journal is_drawer_open={is_drawer_open} />
```

Comment:
what is this about?

Reply:
This is also from upstream by @maryiafrantsava-binary
<img width="1728" alt="Screenshot 2022-07-07 at 2 27 12 PM" src="https://user-images.githubusercontent.com/104334373/177706351-cba0e722-c88f-439f-9c8d-f40c016e6ebf.png">

Reply:
She replaces the line above with <Journal />

Reply:
so in that case if its merged to master it should not be seen in your PR
please merge your branch one more time with master branch
they should disappear

Reply:
She does it [here](https://github.com/binary-com/deriv-app/pull/5793/files#diff-5f7be65cbd2e1e405c5d8a765065fcbd6ab7d35a5ba8111141ad146dbf3d4094)

Reply:
Alright. Thank you